components/button/button.interface.ts

```diff
@@ -10,7 +10,7 @@ export interface IButtonComponentProps {
   type?: EButtonTypes.PRIMARY | EButtonTypes.SECONDARY | EButtonTypes.TERTIARY;
   size?: EButtonSizes.MEDIUM | EButtonSizes.SMALL | EButtonSizes.LARGE;
   label: string;
-  onClickEvent: () => void;
+  onClickEvent?: () => void;
```

Comment:
Why did you make the onClickEvent optional?

Reply:
yes, as a HREF Button should not have any click event.